code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm

```diff
@@ -0,0 +1,195 @@
+		Convert(H)
+
+/mob/living/simple_animal/hostile/abnormality/censored/proc/Convert(mob/living/carbon/human/H)
+	if(!istype(H))
+		return
+	if(!can_act)
+		return
+	can_act = FALSE
+	forceMove(get_turf(H))
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0, WHITE_DAMAGE = 0, BLACK_DAMAGE = 0, PALE_DAMAGE = 0)
+	playsound(src, 'sound/abnormalities/censored/convert.ogg', 65, FALSE, 5)
+	SLEEP_CHECK_DEATH(3)
+	new /obj/effect/temp_visual/censored(get_turf(src))
+	for(var/i = 1 to 3)
+		new /obj/effect/gibspawner/generic/silent(get_turf(src))
+		SLEEP_CHECK_DEATH(5.5)
+	var/mob/living/simple_animal/hostile/mini_censored/C = new(get_turf(src))
+	if(!QDELETED(H))
+		C.desc = "What the hell is this? It shouldn't exist... On the second thought, it reminds you of [H.real_name]..."
+		H.gib()
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.6, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.4, PALE_DAMAGE = 1)
+	can_act = TRUE
+
+/* Work */
+/mob/living/simple_animal/hostile/abnormality/censored/attempt_work(mob/living/carbon/human/user, work_type)
+	if(work_type == "Sacrifice")
+		to_chat(user, "<span class='warning'>You hesitate for a moment...</span>")
+		if(!do_after(user, 3 SECONDS, target = user))
+			to_chat(user, "<span class='warning'>You decide it's not worth it.</span>")
+			return null
+		user.Stun(30 SECONDS)
+		step_towards(user, src)
+		sleep(0.3 SECONDS)
+		step_towards(user, src)
+		new /obj/effect/temp_visual/censored(get_turf(src))
+		sleep(0.3 SECONDS)
+		playsound(src, 'sound/abnormalities/censored/sacrifice.ogg', 45, FALSE, 10)
+		user.death()
+		for(var/i = 1 to 3)
+			new /obj/effect/gibspawner/generic/silent(get_turf(src))
+			sleep(5.4)
+		datum_reference.qliphoth_change(1)
+		QDEL_NULL(user)
+		return null
+	return TRUE
+
+/mob/living/simple_animal/hostile/abnormality/censored/work_complete(mob/living/carbon/human/user, work_type, pe, work_time)
+	..()
+	if(user.sanity_lost)
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/censored/failure_effect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/censored/breach_effect(mob/living/carbon/human/user)
+	..()
+	icon_living = "censored_breach"
+	icon_state = icon_living
+	return
+
+/* The mini censoreds */
+/mob/living/simple_animal/hostile/mini_censored
+	name = "???"
+	desc = "What the hell is this? It shouldn't exist... On the second thought, it reminds you of someone you knew..."
+	icon = 'ModularTegustation/Teguicons/32x32.dmi'
+	icon_state = "censored_mini"
+	icon_living = "censored_mini"
+	faction = list("censored")
+	speak_emote = list("screeches")
+	attack_verb_continuous = "attacks"
+	attack_verb_simple = "attack"
+	attack_sound = 'sound/abnormalities/censored/mini_attack.ogg'
+	/* Stats */
+	health = 600
+	maxHealth = 600
+	obj_damage = 300
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 1.2, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 1)
```

Comment:
At some point I feel like we need to give the manager a "Secret Information" file that holds information for Ordeals and Abnormality "Children" (Bees, CENSOREDs, Perfect Foods, etc)